We do not need to run coverage for this project.

--- pexpect-4.9/requirements-testing.txt.orig
+++ pexpect-4.9/requirements-testing.txt
@@ -1,6 +1,3 @@
 aiounittest; python_version < '3.8'
-coverage
-coveralls
 ptyprocess
 pytest
-pytest-cov
